src/System Application/App/Vs Code Request Management/src/VSCodeRequestManagement.Codeunit.al

```diff
@@ -0,0 +1,147 @@
+﻿namespace System.Integration;
+
+using System.Apps;
+using System.Reflection;
+using System.Utilities;
+
+codeunit 8033 "VS Code Request Management"
```

Comment:
Do we need VS in there? Do we need Management?

Could it be be called something like Code Requests ? or VSCode Requests? or Source Code Requests?

something along those lines

Reply:
VSCode Integration?